src/vivarium_public_health/metrics/disability.py

```diff
@@ -37,67 +41,82 @@ class DisabilityObserver:
                     by_sex: True
 
     """
+
     configuration_defaults = {
-        'metrics': {
-            'disability': {
-                'by_age': False,
-                'by_year': False,
-                'by_sex': False,
-            }
-        }
+        "metrics": {"disability": {"by_age": False, "by_year": False, "by_sex": False,}}
```

Comment:
I don't like how it condenses this into one line.

Reply:
Agreed.

Reply:
I wonder if it would have done what we're expecting if there was another key/value at the level of `"disability"`